projects/plugins/jetpack/tests/e2e/lib/setup-env.js

```diff
@@ -1,272 +0,0 @@
-/* eslint-disable jest/no-export */
```

Comment:
I've noticed that a bunch of functions in this file did not survive the migration :) 

Seems like `circus` environment allows us properly act on failures in hooks and tests, which is great.